datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py

```diff
@@ -104,7 +104,17 @@ def create(ctx, name, integration_type, location, non_interactive, quiet, dry_ru
     root = resolve_path(location) if location else get_root()
     path_sep = os.path.sep
 
-    integration_dir = os.path.join(root, normalize_package_name(name))
+    if integration_type == 'snmp_tile' and name.startswith('snmp_'):
+        name_without_snmp_prefix = name[len('snmp_') :]
+        echo_warning(f'Creating snmp_tile for vendor ({name}) in directory (snmp_{normalize_package_name(name)})')
+        value = click.prompt(f'Did you mean vendor ({name_without_snmp_prefix})? (y/n)', default='y')
+        if value.lower() == 'y':
+            name = name_without_snmp_prefix
```

Comment:
As discussed, IMHO it doesn't feel that great to try to correct the user here (more code to maintain, avoid extra custom snmp code in ddev), I would tend to favour just aborting with an explicit error and ask user to use the command with correct/valid input.

Reply:
I would even skip the warning. If customers (or more probably myself 🤣 ) use something starting with `SNMP`, they would create a `snmp_snmp_something` directory, see it is wrong, undo the change and retrying with a proper vendor name. What do you think?